### PR TITLE
Cleanup application layout

### PR DESCRIPTION
**Pull request by arun1595**
Tasks done:

- Display bootstrap flash messages by removing `in` class with `show` class
- Navbar is displayed in all the pages, hence `@navbar` is useless, removed it
- Fixed failing specs
- Did some cosmetic changes

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
 class StaticPagesController < ApplicationController
   def home
-    @navbar = false
     @is_home_page = true
```
Comment (KevinMulhern) [inline app/controllers/static_pages_controller.rb]: I wonder if we still need this one too?

**Comment by KevinMulhern** [comment]
LGTM 👍 
Nice work @arun1595 😄

File: app/controllers/static_pages_controller.rb (hunk 1 above)
Comment (arun1595) [inline app/controllers/static_pages_controller.rb]: I am not sure. The value of that instance variable decides whether to show `AdminFlashes`. But I have never come across the admin flashes. How are they actually used in the application?

Comment (KevinMulhern) [inline app/controllers/static_pages_controller.rb]: They are flash messages intended to be used for announcements for all users on the site. With the homepage not really being accessible to signed in users now we will want to show them on all pages if we decide to keep that feature.

**Comment by KevinMulhern** [comment]
@arun1595 looks like theres conflicts 🙈 can you fix them please? We will get this :shipit: after that 😄

**Pull request merged**